contribs/emissions/src/main/java/org/matsim/contrib/emissions/EmissionModule.java

```diff
@@ -181,6 +181,11 @@ private void createLookupTables() {
 			addPollutantsToMap(warmPollutants, detailedHbefaWarmTable.keySet());
 		}
 
+		if(emissionConfigGroup.getHbefaConsistencyChecker() == EmissionsConfigGroup.UseHbefaConsistencyChecker.check){
+			logger.info("Checking consistency of hbefa-tables");
+			HbefaConsistencyChecker.checkConsistency(emissionConfigGroup.getDetailedVsAverageLookupBehavior(), avgHbefaWarmTable, detailedHbefaWarmTable, avgHbefaColdTable, detailedHbefaColdTable);
+		}
```

Comment:
maybe give a little warning if no check is conducted?

Reply:
Yes, good idea, @tschlenther !